### DISCUSSION TITLE
Blank screen on ipad after login.

### DISCUSSION THREAD

**Discussion by illispi** [coder/code-server#3045] [Q&A] [answered]
So i followed the guide to setup code-server. I used digital ocean, had my own domain, used caddy as reverse proxy but still blank screen on ipad. Pc and android work fine. There was previous threads on this subject but i didn't find definite answer from them. Any ideas?

**Comment by jsjoeio** [comment, marked as answer]
Hi @illispi - this is a known issue and we should have a fix out soon. Please see: https://github.com/cdr/code-server/discussions/3038

There may be a fix you can try in https://github.com/cdr/code-server/issues/2975
